lib/datadog/enable_core_dumps.rb

```diff
@@ -0,0 +1,40 @@
+# This helper is used to enable core dumps for the current Ruby app. This is useful when debugging native-level
```

Comment:
This has a significant side effect, so it needs `# typed: ignore` otherwise Sorbet will evaluate the file!

Reply:
Fixed in 14afd4d55883f68d7fb9d106b2f46d76014b39d4